spring-petclinic-customers-service/src/main/java/org/springframework/samples/petclinic/customers/web/mapper/Mapper.java

```diff
@@ -0,0 +1,5 @@
+package org.springframework.samples.petclinic.customers.web.mapper;
+
+public interface Mapper<R, E> {
+    E map(E response, R request);
```

Comment:
The return value is not used. Do you think it will be in the future?

Reply:
The return is used in the createOwner() resource . 
The return value is not used in the updateOwner() resource

Reply:
You're right I didn't see that in `createOwner()`. But we could avoid the return value.
It's ok for me like this.